tools/distrib/build_ruby_environment_macos.sh

```diff
@@ -49,8 +49,9 @@ MAKE="make -j8"
 
 for v in 2.5.0 2.4.0 2.3.0 2.2.2 2.1.6 2.0.0-p645 ; do
   ccache -c
-  rake -f $CROSS_RUBY cross-ruby VERSION=$v HOST=x86_64-darwin11
+  rake -f "$CROSS_RUBY" cross-ruby VERSION="$v" HOST=x86_64-darwin11 MAKE="$MAKE"
```

Comment:
Setting `MAKE` here is new... is this needed?

Reply:
That was the bugfix I was alluding to: I think it is needed, otherwise the MAKE= above is useless. The nested script tries to read MAKE environment but it is effectively unset since it is not exported.